templates/21-loggly-base.conf

```diff
@@ -7,5 +7,14 @@ $PrivDropToGroup adm
 # Setup disk assisted queues
 $WorkDirectory /var/spool/rsyslog # where to place spool files
 
-$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %msg%\n"
-$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %msg%\n"
+$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %$!msg%\n"
+$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %$!msg%\n"
+
```

Comment:
we'll have to include the catchall template `LogglyFormat` too - maybe we just move to here for now.